order/client_test.go

```diff
@@ -65,16 +59,15 @@ func TestOrderReturn_RequestParams(t *testing.T) {
 	defer ts.Close()
 
 	// Configure the stripe client to use the ephemeral backend.
-	stripe.SetBackend("api", &stripe.BackendConfiguration{
-		Type:       stripe.APIBackend,
-		URL:        ts.URL + "/v1",
-		HTTPClient: &http.Client{},
+	backend := stripe.GetBackendWithConfig(stripe.APIBackend, &stripe.BackendConfig{
+		URL: ts.URL,
 	})
+	orderClient := Client{B: backend, Key: stripe.Key}
```

Comment:
Instantiating a new client turned out to be a far cleaner way to implement this test anyway (the `defer` to reset the default backend at the top is no longer needed).